src/faq.md

```diff
@@ -79,6 +79,55 @@ outputs:
       outputEval: ${self[0].basename=inputs.otu_table_name; return self;}
 ```
 
+## Referencing a local script
+
+We are going to describe two methods of doing this:
+
+The first method involves [adding the folder containing your scripts to the `PATH` environment variable](https://stackoverflow.com/q/8779951/20123733). This allows you to run the shell script without using `sh` or `bash` commands
+
+
+Start with adding a _shebang_ at the top of your file:
+```
+#!/bin/bash
+```
```

Comment:
Most other examples have blank lines between the code block.

And for code block, I think we should use the `code-block` directive (see other examples for bash... maybe contributing has some info too?) for consistency.